departure.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'rails', '~> 4.2.0'
+  spec.add_runtime_dependency 'rails', '~> 5.1'
```

Comment:
Have you tried with Rails 5.0? We will be upgrading our app to 5.0 in the next few weeks and will need to make this work if you havn't already tried it.

Reply:
I didn't but I guess that the changes you did will also work for Rails 5.0

Reply:
The LHM compatibility later does not work with 5.0. Other than testing it, though, I haven't isolated the fix.

Reply:
This is my attempt to fix it. https://github.com/departurerb/departure/pull/13/commits/6583613a6984f927a263aeef851b52c7d5068eac